src/pages/index.tsx

```diff
@@ -1,21 +1,184 @@
-/* eslint-disable no-unused-vars */
 import * as React from "react";
-import { Link } from "gatsby";
-
 import Layout from "../components/layout";
-import Image from "../components/image";
-import SEO from "../components/seo";
+import { Table as AntTable, Typography } from "antd";
+import { ColumnsType } from "antd/es/table";
+
+interface HardwareData {
+	name: string;
+	identifier: string;
+	repository: string;
+	version: string;
+	license: string;
+	organization: string;
+	language: string;
+	tsdcId: number;
+}
+
+const tableHeaders: ColumnsType<HardwareData> = [
+	{
+		title: "Name",
+		dataIndex: "name",
+		key: "name"
+	},
+	{
+		title: "Identifier",
+		dataIndex: "identifier",
+		key: "identifier",
+		// eslint-disable-next-line react/display-name
+		render: identifier => <a>{identifier}</a>
```

Comment:
What do we use this `render` key for?

Reply:
Render is part of antd Column api, used to return a ReactNode. Here, it's creating an <a> tag from the "identifier" string.

Reply:
Why are we rendering a link for the identifier tag though?

Reply:
Identifier was a link on the original mockup for Explore Data Page.

Reply:
Looking at the mock-up it appears to just be text, the specification doesn't mention it as a link or what it would link to.

Let's remove this and just render text like the other columns